CRITICAL - 2024-02-29 01:49:35 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "Views\data\login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Views\\data\\login.php')
 2 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('Views\\data\\login', [], true)
 3 VENDORPATH\myth\auth\src\Controllers\AuthController.php(412): view('Views\\data\\login', [...])
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(58): Myth\Auth\Controllers\AuthController->_render('Views\\data\\login', [...])
 5 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
CRITICAL - 2024-02-29 03:23:46 --> ErrorException: Undefined array key 1
in VENDORPATH\myth\auth\src\Authentication\Passwords\NothingPersonalValidator.php on line 77.
 1 VENDORPATH\myth\auth\src\Authentication\Passwords\NothingPersonalValidator.php(77): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 1', 'E:\\xampp\\htdocs\\sistem-aps-uho\\vendor\\myth\\auth\\src\\Authentication\\Passwords\\NothingPersonalValidator.php', 77)
 2 VENDORPATH\myth\auth\src\Authentication\Passwords\NothingPersonalValidator.php(26): Myth\Auth\Authentication\Passwords\NothingPersonalValidator->isNotPersonal('kelompokkplppmp2024*', Object(Myth\Auth\Entities\User))
 3 VENDORPATH\myth\auth\src\Authentication\Passwords\PasswordValidator.php(46): Myth\Auth\Authentication\Passwords\NothingPersonalValidator->check('kelompokkplppmp2024*', Object(Myth\Auth\Entities\User))
 4 VENDORPATH\myth\auth\src\Authentication\Passwords\ValidationRules.php(42): Myth\Auth\Authentication\Passwords\PasswordValidator->check('kelompokkplppmp2024*', Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\Validation\Validation.php(327): Myth\Auth\Authentication\Passwords\ValidationRules->strong_password('kelompokkplppmp2024*', null)
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('password', 'password', 'kelompokkplppmp2024*', [...], [...])
 7 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 8 VENDORPATH\myth\auth\src\Controllers\AuthController.php(164): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
CRITICAL - 2024-02-29 05:30:52 --> ErrorException: Undefined array key 1
in VENDORPATH\myth\auth\src\Authentication\Passwords\NothingPersonalValidator.php on line 77.
 1 VENDORPATH\myth\auth\src\Authentication\Passwords\NothingPersonalValidator.php(77): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 1', 'E:\\xampp\\htdocs\\sistem-aps-uho\\vendor\\myth\\auth\\src\\Authentication\\Passwords\\NothingPersonalValidator.php', 77)
 2 VENDORPATH\myth\auth\src\Authentication\Passwords\NothingPersonalValidator.php(26): Myth\Auth\Authentication\Passwords\NothingPersonalValidator->isNotPersonal('kelompokkplppmp2004**', Object(Myth\Auth\Entities\User))
 3 VENDORPATH\myth\auth\src\Authentication\Passwords\PasswordValidator.php(46): Myth\Auth\Authentication\Passwords\NothingPersonalValidator->check('kelompokkplppmp2004**', Object(Myth\Auth\Entities\User))
 4 VENDORPATH\myth\auth\src\Authentication\Passwords\ValidationRules.php(42): Myth\Auth\Authentication\Passwords\PasswordValidator->check('kelompokkplppmp2004**', Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\Validation\Validation.php(327): Myth\Auth\Authentication\Passwords\ValidationRules->strong_password('kelompokkplppmp2004**', null)
 6 SYSTEMPATH\Validation\Validation.php(200): CodeIgniter\Validation\Validation->processRules('password', 'password', 'kelompokkplppmp2004**', [...], [...])
 7 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 8 VENDORPATH\myth\auth\src\Controllers\AuthController.php(164): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
CRITICAL - 2024-02-29 05:44:16 --> Error: Class "false" not found
in VENDORPATH\myth\auth\src\Config\Services.php on line 81.
 1 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
 2 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('activator', [...])
 3 VENDORPATH\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', null)
 4 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator()
 5 SYSTEMPATH\Common.php(1008): CodeIgniter\Config\BaseService::__callStatic('activator', [])
 6 VENDORPATH\myth\auth\src\Controllers\AuthController.php(184): service('activator')
 7 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
ERROR - 2024-02-29 06:37:47 --> Email: sendWithMail throwed ErrorException: mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() in E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php:1761
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mail(): Failed ...', 'E:\\xampp\\htdocs...', 1761)
#1 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1761): mail('wahyunahdaputra...', '=?UTF-8?Q?Activ...', 'This is a multi...', 'Date: Thu, 29 F...', '-f wahyunahdapu...')
#2 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1693): CodeIgniter\Email\Email->sendWithMail()
#3 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1587): CodeIgniter\Email\Email->spoolEmail()
#4 E:\xampp\htdocs\sistem-aps-uho\vendor\myth\auth\src\Authentication\Activators\EmailActivator.php(32): CodeIgniter\Email\Email->send()
#5 E:\xampp\htdocs\sistem-aps-uho\vendor\myth\auth\src\Controllers\AuthController.php(185): Myth\Auth\Authentication\Activators\EmailActivator->send(Object(Myth\Auth\Entities\User))
#6 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#7 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#8 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\xampp\htdocs\sistem-aps-uho\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs...')
#11 {main}
ERROR - 2024-02-29 06:37:47 --> Email: Unable to send email using PHP mail(). Your server might not be configured to send mail using this method.
ERROR - 2024-02-29 06:37:47 --> 
ERROR - 2024-02-29 07:07:57 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2024-02-29 07:07:57 --> 220 smtp.gmail.com ESMTP fa13-20020a17090af0cd00b00298f2ad430csm688882pjb.0 - gsmtp

hello: 250-smtp.gmail.com at your service, [103.161.192.141]
250-SIZE 35882577
250-8BITMIME
250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

Failed to authenticate password. Error: 534-5.7.9 Application-specific password required. For more information, go to
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor fa13-20020a17090af0cd00b00298f2ad430csm688882pjb.0 - gsmtp

ERROR - 2024-02-29 07:15:06 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2024-02-29 07:15:06 --> 220 smtp.gmail.com ESMTP y6-20020a634b06000000b005ce472f2d0fsm521273pga.66 - gsmtp

hello: 250-smtp.gmail.com at your service, [103.161.192.141]
250-SIZE 35882577
250-8BITMIME
250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

Failed to authenticate password. Error: 534-5.7.9 Application-specific password required. For more information, go to
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor y6-20020a634b06000000b005ce472f2d0fsm521273pga.66 - gsmtp

ERROR - 2024-02-29 07:48:13 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2024-02-29 07:48:13 --> 220 smtp.gmail.com ESMTP t26-20020a62d15a000000b006e04c3b3b5asm684981pfl.175 - gsmtp

hello: 250-smtp.gmail.com at your service, [103.161.192.141]
250-SIZE 35882577
250-8BITMIME
250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

Failed to authenticate password. Error: 534-5.7.9 Application-specific password required. For more information, go to
534 5.7.9  https://support.google.com/mail/?p=InvalidSecondFactor t26-20020a62d15a000000b006e04c3b3b5asm684981pfl.175 - gsmtp

CRITICAL - 2024-02-29 07:51:35 --> Error: Class "false" not found
in VENDORPATH\myth\auth\src\Config\Services.php on line 81.
 1 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
 2 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('activator', [...])
 3 VENDORPATH\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', null)
 4 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator()
 5 SYSTEMPATH\Common.php(1008): CodeIgniter\Config\BaseService::__callStatic('activator', [])
 6 VENDORPATH\myth\auth\src\Controllers\AuthController.php(184): service('activator')
 7 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
CRITICAL - 2024-02-29 07:53:35 --> Error: Call to a member function send() on null
in VENDORPATH\myth\auth\src\Controllers\AuthController.php on line 185.
 1 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
CRITICAL - 2024-02-29 08:05:07 --> Error: Class "false" not found
in VENDORPATH\myth\auth\src\Config\Services.php on line 81.
 1 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator(Object(Myth\Auth\Config\Auth), false)
 2 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('activator', [...])
 3 VENDORPATH\myth\auth\src\Config\Services.php(74): CodeIgniter\Config\BaseService::getSharedInstance('activator', null)
 4 SYSTEMPATH\Config\BaseService.php(258): Myth\Auth\Config\Services::activator()
 5 SYSTEMPATH\Common.php(1008): CodeIgniter\Config\BaseService::__callStatic('activator', [])
 6 VENDORPATH\myth\auth\src\Controllers\AuthController.php(184): service('activator')
 7 SYSTEMPATH\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs\\sistem-aps-uho\\public\\index.php')
ERROR - 2024-02-29 08:05:44 --> Email: sendWithSmtp throwed ErrorException: fsockopen(): php_network_getaddresses: getaddrinfo for smtp.example.com failed: No such host is known.  in E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php:1895
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fsockopen(): ph...', 'E:\\xampp\\htdocs...', 1895)
#1 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1895): fsockopen('ssl://smtp.exam...', 465, NULL, NULL, 60)
#2 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1808): CodeIgniter\Email\Email->SMTPConnect()
#3 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1693): CodeIgniter\Email\Email->sendWithSmtp()
#4 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Email\Email.php(1587): CodeIgniter\Email\Email->spoolEmail()
#5 E:\xampp\htdocs\sistem-aps-uho\vendor\myth\auth\src\Authentication\Activators\EmailActivator.php(32): CodeIgniter\Email\Email->send()
#6 E:\xampp\htdocs\sistem-aps-uho\vendor\myth\auth\src\Controllers\AuthController.php(185): Myth\Auth\Authentication\Activators\EmailActivator->send(Object(Myth\Auth\Entities\User))
#7 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\CodeIgniter.php(941): Myth\Auth\Controllers\AuthController->attemptRegister()
#8 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#9 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\xampp\htdocs\sistem-aps-uho\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 E:\xampp\htdocs\sistem-aps-uho\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('E:\\xampp\\htdocs...')
#12 {main}
ERROR - 2024-02-29 08:05:44 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2024-02-29 08:05:44 --> 
